build-files/config.json

```diff
@@ -0,0 +1,83 @@
+{
+  "GOOGLE_OAUTH_CLIENT_ID": "",
```

Comment:
must be overridden in **secrets.json**. I will add a template file **secrets.template.json** with all required properties that should be filled in for **local** development.

Reply:
I think it's nice to leave the same comment in the code.